README.md

```diff
@@ -2,54 +2,37 @@
 
 [![Build Status](https://devextreme-ci.devexpress.com/api/badges/DevExpress/DevExtreme/status.svg?branch=20_1)](https://devextreme-ci.devexpress.com/DevExpress/DevExtreme)
 
-Voted the best cross-platform component suite containing everything you need to create responsive web apps for touch devices and traditional desktops: data grid, interactive charts, data editors, navigation and multi-purpose widgets that are designed to look great and provide powerful functionality in any browser.
+Enterprise-ready UI component suites for Angular, React, Vue, and jQuery. Everything you need to create responsive web apps for touch devices and traditional desktops: data grid, interactive charts, data editors, navigation and multi-purpose widgets that are designed to look great and provide powerful functionality in any browser.
```

Comment:
These are incomplete sentences. I rewrote them.
```suggestion
DevExtreme is an enterprise-ready UI component suite for Angular, React, Vue, and jQuery. It is everything you need to create responsive web apps for touch devices and traditional desktops - data grid, interactive charts, data editors, navigation and multi-purpose widgets - controls that are designed to look great and provide powerful functionality in any browser.
```

Reply:
```suggestion
DevExtreme is a set of enterprise-ready UI component suites for Angular, React, Vue, and jQuery. It is everything you need to create responsive web apps for touch devices and traditional desktops - data grid, interactive charts, data editors, navigation and multi-purpose widgets - controls that are designed to look great and provide powerful functionality in any browser.
```

Reply:
DevExtreme is a set of enterprise-ready UI component suites for Angular, React, Vue, and jQuery. It is everything you need to create responsive web apps for touch devices and traditional desktops: data grid, interactive charts, data editors, navigation and multi-purpose widgets. These controls are designed to look great and to provide powerful functionality in any browser.